tests/unit/http/test_mixins.py

```
@@ -8,11 +8,15 @@
 from mpt_api_client import RQLQuery
 from mpt_api_client.exceptions import MPTAPIError
 from mpt_api_client.http import AsyncService, Service
-from mpt_api_client.http.mixins import (
+from mpt_api_client.http.mixins import (  # noqa: WPS235
```

Ruff (0.14.8)

Unused `noqa` directive (unknown: `WPS235`)

Remove unused `noqa` directive

(RUF100)